product-library.adoc

```diff
@@ -10,3 +10,60 @@ https://products.izettle.com
 https://products.izettle.com/swagger
 
 ---
+### Example Requests
+#### Create product
+`POST https://products.izettletest.com/organizations/self/products`
+
+https://products.izettle.com/swagger/#!/products/createProduct
+```json
+{
+  "uuid": "d62f7bb0-2728-11e6-85b5-dd108c223139",
+  "categories": ["shoes","clothing"],
+  "name": "ProductName",
+  "description": "Description of product",
```

Comment:
metadescriptioooon! :boom:

Reply:
Booom! I heard you like descriptions so I put some descriptions in the description

Reply:
Please add more cowbell to description 🐮🔔